src/main/java/org/recommendation/data/GenreDataInitializer.java

```diff
@@ -0,0 +1,36 @@
+package org.recommendation.data;
+
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.recommendation.log.Logger;
+import org.recommendation.log.SoutLogger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GenreDataInitializer extends DataInitializer {
+    public static HashMap<String, String> GenereMap = new HashMap<>();
+    private Logger logger = new SoutLogger();
+
+    @Override
+    public Map<String, String[]> readAndCleanData(final BufferedReader br) {
+        String line;
+        try {
+            while ((line = br.readLine()) != null) {
+                String[] data = line.split("\\|");
```

Comment:
splitter should be part of method arguments so that we can provide more flexibility